FTheoryToolsInOSCAR.ipynb

```diff
@@ -860,32 +1124,1130 @@
+   "source": [
+    "The following method returns the vertices of the polytope the the base of the F-Theory QSM is\n",
+    "build from. Note that those vertices are normalized according to the Polymake standard\n",
+    "to rational numbers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "id": "22708169-4ca1-478d-9848-4f96e5b64e42",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4-element Vector{Vector{QQFieldElem}}:\n",
+       " [-1, -1, -1]\n",
+       " [1, -1//2, -1//2]\n",
+       " [-1, 2, -1]\n",
+       " [-1, -1, 5]"
+      ]
+     },
+     "execution_count": 52,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vertices(qsm_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28e98253-980d-44aa-88bd-9c9eaccf202c",
+   "metadata": {},
+   "source": [
+    "Of the 3-dimensional reflexive polytope that the base of this F-theory model is build from,\n",
+    "the following method returns the index within the Kreuzer-Skarke list:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "id": "38a90e0b-08ce-4bc8-b0fb-6706ecc58234",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "4"
+      ]
+     },
+     "execution_count": 53,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "polytope_index(qsm_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a58994c9-3562-47e3-b158-7a1a0b93705b",
+   "metadata": {},
+   "source": [
+    "For a 3-dimensional reflexive polytope in the Kreuzer-Skarke list, the list of\n",
+    "full (sometimes also called fine), regular, star triangulations can be extremely\n",
+    "large. Consequently, one may wonder if the triangulations can be enumerated in a\n",
+    "somewhat reasonable time (say 5 minutes on a personal computer). The following method tries\n",
+    "to provide an answer to this. It returns `true` if one should expect a timly response\n",
+    "to the atttempt to enumerate all (full, regular, star) triangulations. Otherwise, this\n",
+    "method returns `false`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 54,
+   "id": "4618d3c4-18bc-4d30-8420-2528d1b8ad07",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "true"
+      ]
+     },
+     "execution_count": 54,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "has_quick_triangulation(qsm_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a648b76-b140-44e6-829c-399f7a0134c9",
+   "metadata": {},
+   "source": [
+    "In order to enumerate the number of full, regular, star triangulations of a\n",
+    "3-dimensional reflexive polytope, it is possible to first find the corresponding\n",
+    "triangulations of all facets of the polytope [HT17](@cite). A first indication for\n",
```

Comment:
This @cite will not work here. Please insert the link directly. In this case, I would expect something "cf. work by Halverson and Tian in 2017, see *insert link*". Or you even say the title of the paper.

Likewise with the other instances of citations. I believe to have added about 4 to 5 papers.

Details of these papers are in https://github.com/oscar-system/Oscar.jl/blob/master/docs/oscar_references.bib.